### PR TITLE
Add Pickle Finance

### PR DESCRIPTION
**Pull request by picklerz**
Pickle Finance is a protocol aiming to bring stablecoins closer to their pegs by use of yVault like strategies and farming incentives. Learn more at the links below.

Site: http://pickle.finance/
Twitter: https://twitter.com/picklefinance

### CODE REVIEW

**Review by e-asia — APPROVED** [review]

File: spaces/pickle/skin.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    color: white !important;
+
+    &:hover {
+      color: #53ffe2 !important;
+    }
+  }
+
+  .text-white {
+    color: white !important;
+  }
```
Comment (e-asia) [inline spaces/pickle/skin.scss:35]: @picklerz Unfortunately we can't take custom styles at the moment you should just use style variables

**Comment by 0xpenguin** [comment]
@e-asia removed the custom styles

File: spaces/pickle/index.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "pickle",
+  "symbol": "PICKLE C",
+  "name": "PICKLEs In The Citadel",
```
Comment (bonustrack) [inline spaces/pickle/index.json:4]: Can you chose a shorter name? This will break the interface on small screen otherwise.

**Comment by 0xpenguin** [comment]
@bonustrack done!

**Comment by picklerz** [comment]
@e-asia @bonustrack the changes have been made, merge when ready! thanks for reviewing :)

**Pull request merged**